dependencies/lib/perl/Transcript.pm

```diff
@@ -68,6 +68,8 @@ sub get_feature {
 
 	return 'intergenic' if($bp > $self->end && $strand eq '+');
 	return 'intergenic' if($bp < $self->start && $strand eq '-');
+	return 'intergenic' if($bp < $self->start && $strand eq '+');#Tian
```

Comment:
Have you checked with Michael E.? I'm not sure if this code is used elsewhere, but we want to make sure this does not cause issues in anything else that assumes the current method.

Reply:
@mnedmonson - This is `common-perllib-genome/perllib/Transcript.pm`. I'll mirror the changes to SVN once this is ready, but I wanted to make sure you were aware and had an opportunity to review this change as it is your code and may affect things other than CICERO.

Reply:
Sorry for the delay, I'm signed with github w/my personal email where the notifications were buried.  Actually while I did write a lot of stuff in common-perllib-genome this module isn't one of them, it looks like it was checked into svn originally by Yongjin.

Maybe a 4th line for the other "-" case as well?  Not sure about the transcript format, if it's refSeq or something else, depending on the details it may be that a simpler start/end boundary check might work regardless of strand (?)

Reply:
Thanks, Michael. I should've commented one line down as Liqing has the 4th case there. Thanks for taking a look, I saw the path and thought this was yours. I should've checked the history.